package.json

```diff
@@ -5,9 +5,9 @@
   "license": "GPL-3.0",
   "scripts": {
     "webpack": "NODE_ENV=production webpack -p --bail",
-    "start": "webpack-dev-server --compress",
-    "start:noc": "webpack-dev-server",
-    "startstage": "API_HOST=http://hchung.westus.cloudapp.azure.com:5000 npm start",
+    "start": "webpack-dashboard -- webpack-dev-server --inline --compress",
+    "start:noc": "webpack-dashboard -- webpack-dev-server --inline",
+    "startstage": "NODE_ENV=production API_HOST=http://hchung.westus.cloudapp.azure.com:5000 webpack-dev-server --compress",
```

Comment:
we should do this like `start:stage`. It's a nice convention for scripts to use the colon to extend functionality.